doc/releases/release-notes-3.7.rst

```diff
@@ -377,6 +377,18 @@ Drivers and Sensors
 * Flash
 
   * Added support for Ambiq Apollo3 series.
+  * Added support for multi-instance of SPI NOR driver (spi_nor.c).
+  * Added preliminary support for non-erase devices with introduction of
+    device capabilities to c:struct:`flash_parameters` and utility function
+    c:func:`flash_params_get_erase_cap` that allows to obtain erase type
+    provided by a device; added c:macro:`FLASH_ERASE_C_EXPLICIT`, which is
+    currently only supported erase type and is set by all flash devices.
+  * Added c:func:`flash_flatten` function that can be used on devices
+    with and without erase when erase is used not for preparing device for
```

Comment:
If there something missing before that "when"?
(I can't really parse this sentence)

Reply:
Improved, hopefully, the description.